extensions/wikia/Wall/WallHooksHelper.class.php

```diff
@@ -1477,18 +1477,30 @@ static public function onWatchArticle( &$user, &$article ) {
 		return true;
 	}
 
+	/**
+	 * @param User $user
+	 * @param Article $article
+	 *
+	 * @return bool
+	 */
 	static public function onUnwatchArticle( &$user, &$article ) {
-		$app = F::app();
 		$title = $article->getTitle();
 
-		if ( !empty( $app->wg->EnableWallExt ) && static::isWallMainPage( $title ) ) {
```

Comment:
This hook wouldn't be registered if ```EnableWallExt``` wasn't set and we need this thing to fire even when this isn't a wall main page, because Forums.